### PR TITLE
[MINDEXER-135] Fix CLI shading

### PR DESCRIPTION
**Pull request by cstamas**
And overall fix shading warnings.

### CODE REVIEW

File: indexer-cli/src/main/java/org/apache/maven/index/cli/NexusIndexerCli.java [hunk 1]
```diff
@@ -176,7 +176,7 @@ public CommandLine parse( String[] args )
     @Override
     public String getPomPropertiesPath()
     {
-        return "META-INF/maven/org.sonatype.nexus/nexus-indexer/pom.properties";
+        return "META-INF/maven/org.apache.maven.indexer/indexer-core/pom.properties";
```
Comment (cstamas) [inline indexer-cli/src/main/java/org/apache/maven/index/cli/NexusIndexerCli.java:179]: java -jar indexer-cli was not working due this:
```
cstamas@Urnebes ~/Worx/apache-maven/maven-indexer  (master *)$ java -jar indexer-cli/target/indexer-cli-6.1.0-SNAPSHOT.jar -v
Unable determine version from JAR file.
```
This fixes it (as after code donation the coordinates changed, but seems this was neglected)

File: indexer-cli/pom.xml [hunk 2]
```diff
@@ -91,6 +97,7 @@ under the License.
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-webapp</artifactId>
+      <scope>test</scope>
```
Comment (cstamas) [inline indexer-cli/pom.xml:100]: PR https://github.com/apache/maven-indexer/pull/165 missed to set scope (is test)

File: indexer-cli/src/main/java/org/apache/maven/index/cli/NexusIndexerCli.java (hunk 1 above)
Comment (gnodet) [reply indexer-cli/src/main/java/org/apache/maven/index/cli/NexusIndexerCli.java:179]: There isn't any test that exercise that ?

**Review by gnodet — APPROVED** [review]

File: indexer-cli/src/main/java/org/apache/maven/index/cli/NexusIndexerCli.java (hunk 1 above)
Comment (cstamas) [reply indexer-cli/src/main/java/org/apache/maven/index/cli/NexusIndexerCli.java:179]: Sadly no :disappointed:

**Comment by jira-importer** [comment]
Resolve #536

**Comment by jira-importer** [comment]
Resolve #536

**Pull request merged**